packages/common-utils/src/renderChartConfig.ts

```diff
@@ -790,7 +790,7 @@ function translateMetricChartConfig(
                any(Attributes) OVER (ROWS BETWEEN 1 PRECEDING AND 1 PRECEDING) AS PrevAttributes,
                IF(AggregationTemporality = 1,
                   Value,IF(Value - PrevValue < 0 AND Attributes = PrevAttributes,Value,
-                      IF(Attributes != PrevAttributes, 0, Value - PrevValue))) as Rate
+                      IF(cityHash64(Attributes) != cityHash64(PrevAttributes), 0, Value - PrevValue))) as Rate
```

Comment:
I think we want to actually run this on the subselect first, otherwise we'll be rehashing a lot more than we need to.